sdk/python/examples/psycopg2_opentelemetry.py

```diff
@@ -0,0 +1,39 @@
+"""This is an example that shows Approzium Opentelemetry integration. It also
+integrates with a Jaeger service to export and view generated traces.
+"""
+from opentelemetry import trace
+from opentelemetry.ext import jaeger
+from opentelemetry.ext.psycopg2 import Psycopg2Instrumentor
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
+
+import approzium
+import approzium.opentelemetry
+from approzium.psycopg2 import connect
+
+auth = approzium.AuthClient("authenticator:6001")
+approzium.default_auth_client = auth
+
+trace.set_tracer_provider(TracerProvider())
+
+jaeger_exporter = jaeger.JaegerSpanExporter(
+    service_name="approzium_service", agent_host_name="jaeger", agent_port=6831
+)
+
+trace.get_tracer_provider().add_span_processor(
+    BatchExportSpanProcessor(jaeger_exporter)
```

Comment:
This is interesting. In the Go library, on the server side, you have to name spans. [Here's an example.](https://github.com/cyralinc/approzium/compare/server-side-tracing#diff-55038164729d27ab2f2b2be0f926c1f8R36) I'm still working on that branch and I'm not 100% sure if I'm _using_ spans the **right** way, I'm planning to test and consult more docs. _Should_ we be creating named spans here? And if on the server side we name spans at the request level, would we want to do that in the Python SDK too?

Reply:
I don't think, at least here. I am starting to realize that there are two flavors of tracing. The first is the automatic span generation that the Opentelemetry DBAPI layer adds. The second is us generating spans for our calls to Authenticator and other services. I did *not* implement that here, and I didn't consider if it is what we want or not. In the case of the latter, we would create our own spans. In the first, we simply add some tags to already existing spans, not worrying about when/how they are created.